2_process_sw_gw_site_data.R

```diff
@@ -35,25 +35,27 @@ p2_sw_gw_site_targets_list <- list(
              
   ),
   
```

Comment:
Would this script be more properly named `2_process_site_data.R`? That would keep it open to including WQ processing here